resources/views/email/signup/user.blade.php

```diff
@@ -4,10 +4,18 @@
         Hello **{{ $user["first_name"] }} {{ $user["last_name"] }},**
 
         You have successfully been registered on the SpringVerse platform.
-        Please find your login credentials below:
 
-        * Email - {{ $user['email'] }}
-        * Default Password - {{ $defaultPassword }}
+        @if($source == "BACKEND")
+            {
+            Please find your login credentials below:
+
+            * Email - {{ $user['email'] }}
+            * Default Password - {{ $defaultPassword }}
+            }
+        @else{
+
```

Comment:
Where is the text for emails sent to users that register online? 

Note that for online users, do not show their password in the email